src/ConceptHtmlRouteProvider.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\rdf_skos;
+
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Entity\Routing\AdminHtmlRouteProvider;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Provides routes for SKOS Concept entities.
+ */
+class ConceptHtmlRouteProvider extends AdminHtmlRouteProvider {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getRoutes(EntityTypeInterface $entity_type): RouteCollection {
+    $collection = parent::getRoutes($entity_type);
+
+    $routes = [
+      'entity.skos_concept.edit_form',
+      'entity.skos_concept.delete_form',
+      'entity.skos_concept.canonical',
+    ];
+    foreach ($routes as $name) {
+      if ($route = $collection->get($name)) {
```

Comment:
It's not very clear this syntax, could we have?
`if (null !== ($route = $collection->get($name)))`

Reply:
Implicit type casting is fine here and it's a common practice.